### ISSUE TITLE
`bad-continuation` recognized ijn 2.13.9 but not in 2.14 

### ISSUE THREAD

**Issue by rafaellehmkuhl** [PyCQA/pylint#6809]
### Question

I have [a CI running with pylint](https://github.com/bluerobotics/BlueOS-docker/runs/6708805888?check_suite_focus=true#step:5:18) and I disable the `bad-continuation` check in pyproject.toml:
```toml
[tool.pylint]
    [tool.pylint.master]
    enable = "all"
    disable = [
        "attribute-defined-outside-init",
        "bad-continuation", # Black takes care of that for us
        "broad-except",
        "duplicate-code",
        "import-error",
        "inconsistent-return-statements",
        "invalid-name",
        "line-too-long", # We already have this in black.
        "locally-disabled", # We know what we are doing, necessary to load plugins in runtime
        "logging-fstring-interpolation", # We want a good-to-read logging interface
        "missing-class-docstring",
        "missing-function-docstring",
        "missing-module-docstring",
        "no-member", # Some members are created in runtime, pylint fails to detect such cases
        "no-name-in-module", # Names can be generated in runtime
        "protected-access", # Necessary for internal tests
        "suppressed-message",
        "too-few-public-methods", # We know what kind of architecture we want
        "too-many-public-methods", # Wrappers will have multiple public methods
        "useless-super-delegation", # Fails to check it properly
        "wildcard-import", # Necessary to locate installed plugins
        "wrong-import-position",
        ]

    [tool.pylint.miscellaneous]
    notes = [ "FIXME" ] # TODO is removed for internal development
```
With pylint 2.13.9 everything was fine, but with the update to 2.14.0 I started receiving this:
```bash
************* Module /home/runner/work/BlueOS-docker/BlueOS-docker/pyproject.toml
pyproject.toml:1:0: E0012: Bad option value for --disable. Don't recognize message bad-continuation. (bad-option-value)
```

### Documentation for future user

I couldn't find any information regarding the `bad-continuation` check (or changes to it) in the docs. Is this the case where it was removed some versions ago and the 2.14.0 refactor to the `bad-option-value` is now warning about it?
